### DISCUSSION TITLE
User Defined Table Function

### DISCUSSION THREAD

**Discussion by gangtao** [timeplus-io/proton#727] [Ideas]
I am looking for a UDF that generate some tables

For example:
`select * from generator(talk)`

I would like to call LLM to generate some structure data as my table input

Any comments?

**Comment by yokofly** [comment]
this title is confusing :)
Previously I thought this was just `select * from js_udf(xxx) from table(stream_name)`.

**Reply by jovezhong** [reply]
Yes, I changed the title to User Defined Table Function
